support-frontend/assets/pages/paper-subscription-landing/components/hero/discountCopy.jsx

```diff
@@ -14,11 +14,11 @@ const discountCopy = (discountPercentage: number): DiscountCopy => (
   discountPercentage > 0 ?
     {
       roundel: ['Save up to', `${discountPercentage}%`, 'a year'],
-      heading: `Save up to ${discountPercentage}% a year with a subscription`,
+      heading: [`Save up to ${discountPercentage}% a year with a for a year on`, <br />, 'The Guardian and The Observer'],
```

Comment:
This copy isn't right I don't think

Reply:
Yes, it's definitely funny!

There's also a note on the trello card to have a discussion with Joel and Gus about keeping the campaign text changes, which was what Joel had requested. However, we need to check with Gus that he is happy with these as ongoing changes.

![Screen Shot 2020-02-11 at 16 51 44](https://user-images.githubusercontent.com/16781258/74258812-d3f1fb80-4cee-11ea-8db8-559412f483e2.png)

Reply:
But obvs changing the percentages as required!

Reply:
I have reinstated the campaign text here.